src/Discord.Net.WebSocket/Net/CacheProviders/EntityConverter.cs

```diff
@@ -77,22 +65,22 @@ private static (object value, int length) Decode(Type type, byte[] data, ref int
             }
             else if (type.IsClass && !type.FullName.StartsWith("System."))
             {
-                var args = new object[] { data.Skip(indx), indx };
-                var m = typeof(EntityConverter)
+                var args = new object[] { data.Skip(index), index };
+                var obj = typeof(EntityConverter)
                             .GetMethod(nameof(EntityConverter.Decode), BindingFlags.NonPublic | BindingFlags.Static)
                             .MakeGenericMethod(type)
                             .Invoke(null, args);
 
-                var l = (int)args[1];
-                indx += l;
-                return (m, l);
+                var firstArgsLength = (int)args[1];
```

Comment:
this would be a ref so i would just call it lengthRef

Reply:
It has been done.